db/migrations/2025_04_03_121425_add_time_units_to_study.py

```diff
@@ -0,0 +1,22 @@
+import sqlalchemy as sql
+
+
+def up(conn):
+    query = """
+        ALTER TABLE Study
+        ADD COLUMN timeUnits VARCHAR(100) COLLATE utf8mb4_bin
+    """
+    conn.execute(sql.text(query))
```

Comment:
Time units allow us to easily convert between different studies' measurement time points